t/selenium.js

```diff
@@ -596,8 +596,10 @@ async function runCommands(commands, t) {
       'psql',
       [
         ...hostPort,
-        '-U', systemDb.user,
-        '-c', `
+        '-U',
+        systemDb.user,
+        '-c',
+        `
```

Comment:
It doesn't hurt readability here so seems fine, though in general I'd ignore the rule for command line flags since it's much more logical to put the flag on the same line as the value